MBO/src/mbo/view/MovingBlockOverlayController.java

```diff
@@ -93,24 +98,23 @@
 
   @Override
   public void initialize(URL location, ResourceBundle resources) {
+    setMBOMode(mboMode);
```

Comment:
this should initialize as false because we're starting fixed-block mode, ie. setMboMode(false);